python/res/job_queue/job_manager.py

```diff
@@ -126,6 +127,7 @@ class JobManager(object):
 
     DEFAULT_UMASK =  0
     sleep_time    =  10  # Time to sleep before exiting the script - to let the disks sync up.
+    MEMORY_POLL_PERIOD = 0.5  # Time between memory polling of job process
```

Comment:
is this in seconds? I assume users are mostly interested in a small selection of jobs (rms, eclipse, etc) which run for at least for a few minutes, but can go up to hours or days

Reply:
Yes, I was wondering what an appropriate value would be for the poll period, adjust it to 5 seconds?